guides/manual-installation.md

```diff
@@ -340,364 +304,323 @@ journalctl -fu trento-web
-```
+1. Obtain a certificate and configure Nginx with Certbot:
+    > **Note:** Replace `example.com` with your domain. For more information, visit [Certbot instructions for Nginx](https://certbot.eff.org/instructions?ws=nginx&os=leap)
 
-> Note: Certbot certificates last for 90 days. Refer to the above link for details on how to renew periodically.
+    ```bash
+    certbot --nginx -d example.com -d www.example.com
+    ```
 
-#### Install and configure nginx
+    > **Note:** Certbot certificates last for 90 days. Refer to the above link for details on how to renew periodically.
 
-**Step 1**: Install nginx package:
+## Install and configure NGINX
 
-```bash
-zypper install nginx
-```
+1. Install NGINX package:
 
-**Step 2**: Add firewalld exceptions for HTTP and HTTPS **(ONLY if firewalld is running)**:
+    ```bash
+    zypper install nginx
+    ```
 
-```bash
-firewall-cmd --zone=public --add-service=https --permanent
-firewall-cmd --zone=public --add-service=http --permanent
-firewall-cmd --reload
-```
+1. Add firewalld exceptions for HTTP and HTTPS **(ONLY if firewalld is running)**:
 
-**Step 3**:Start and enable nginx:
+    ```bash
+    firewall-cmd --zone=public --add-service=https --permanent
+    firewall-cmd --zone=public --add-service=http --permanent
+    firewall-cmd --reload
+    ```
+1. Start and enable nginx:
 
-```bash
-systemctl enable --now nginx
-```
+    ```bash
+    systemctl enable --now nginx
+    ```
 
-**Step 4**: Create a configuration file for Trento:
+1. Create a configuration file for Trento:
 
-```bash
-vim /etc/nginx/conf.d/trento.conf
-```
+    ```bash
+    vim /etc/nginx/conf.d/trento.conf
+    ```
 
-Add the following configuration
+1. Add the following configuration to `/etc/nginx/conf.d/trento.conf`:
 
-```bash
-server {
-    # Redirect HTTP to HTTPS
-    listen 80;
-    server_name trento.example.com;
-    return 301 https://$host$request_uri;
-}
-
-server {
-    # SSL configuration
-    listen 443 ssl;
-    server_name trento.example.com;
-
-    ssl_certificate /etc/ssl/certs/trento.crt;
-    ssl_certificate_key /etc/ssl/private/trento.key;
-
-    ssl_protocols TLSv1.2 TLSv1.3;
-    ssl_ciphers 'ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384';
-    ssl_prefer_server_ciphers on;
-    ssl_session_cache shared:SSL:10m;
-
-    # Wanda rule
-    location ~ ^/(api/checks|api/v1/checks|api/v2/checks)/  {
-        allow all;
-
-        # Proxy Headers
-        proxy_http_version 1.1;
-        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
-        proxy_set_header Host $http_host;
-        proxy_set_header X-Cluster-Client-Ip $remote_addr;
-
-        # The Important Websocket Bits!
-        proxy_set_header Upgrade $http_upgrade;
-        proxy_set_header Connection "upgrade";
-
-        proxy_pass http://localhost:4001;
+    ```bash
```

Comment:
That's not a bash file. Not sure which format it have, but maybe use a different entry. Or remove the entry.

Reply:
Removed